sycl/test-e2e/bindless_images/array/read_sampled_array.cpp

```diff
@@ -1,5 +1,10 @@
+// REQUIRES: aspect-ext_oneapi_bindless_images
 // REQUIRES: aspect-ext_oneapi_image_array
 
+// UNSUPPORTED: level_zero
+// UNSUPPORTED-INTENDED: Linear sampling rounds differently on Intel devices
+// compared to what is expected.
```

Comment:
Slight nit, but "expected" is not really defined. The "expected" value is different for different devices. I would specify that we are comparing to CUDA rounding here, and that this rounding needs to be accounted for, e.g.:

> // UNSUPPORTED-INTENDED: Linear sampling rounds differently on Intel devices
// compared to CUDA. This difference in rounding mode needs to be accounted for.

Reply:
Updated to very slightly modified version of your suggestion.